frontend/.env.example

```diff
@@ -100,6 +100,9 @@ INTEROP_STATUS_CHECK_API_BASE_URI=
 # (optional; default: undefined)
 INTEROP_STATUS_CHECK_API_SUBSCRIPTION_KEY=
 
+# CDCP back end api
+CDCP_API_BASE_URI=https://api.example.com
```

Comment:
This URL has already been used for the interop endpoint, so you should choose something else. Otherwise, it might cause issues when mocking.

Reply:
You might consider opening a separate PR that changes the interop example URL so something a little more specific (ie: `api.interop.example.com`), then rebase this PR on that one.